cpu/native/startup.c

```diff
@@ -106,6 +109,7 @@ static const char short_opts[] = ":hi:s:deEoc:"
 #endif
 #ifdef MODULE_SOCKET_ZEP
     "z:"
+    "Z:"
```

Comment:
```suggestion
    "U:"
```

Reply:
good catch!

Reply:
Welp, it was your test description that caught it ;-)